src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -541,6 +543,44 @@ private static bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 			return true;
 		}
 
+		private bool ProcessCssFile(IRequestInfo info, string localPath)
+		{
+			if  (CurrentBook == null) return false;
+
+			var fileName = localPath;
+			var pos = fileName.LastIndexOfAny(new[] { '\\', '/' });
+			if (pos > -1) fileName = fileName.Substring(pos + 1);
+
+			// try to find the css file
+			var path = CurrentBook.GetFileLocator().LocateFile(fileName);
+
+			// we want these stylesheets to come from the book folder
+			if (path.Contains("languageDisplay.css"))
```

Comment:
I think filename== would be a better test here. I don't know of anything except that exact file that we want this rule to apply to, so why make it more general and less efficient?